data/json/furniture_and_terrain/furniture-appliances.json

```
@@ -604,6 +604,35 @@
       ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_melchior_unit",
+    "name": "supercomputer",
+    "looks_like": "f_server",
+    "description": "Hanging platters assembled into an inverted conical, their positions kept stable by dense threading of copper cabling.",
```

```suggestion
    "description": "Hanging platters assembled into an inverted cone, their positions kept stable by dense threading of copper cabling.",
```
Either that or there's a word missing after conical, no?